src/providers/Social/pool.sdk.ts

```diff
@@ -0,0 +1,84 @@
+import { PollRepository } from '@amityco/ts-sdk-react-native';
```

Comment:
i'm not sure, why do we need to wrap these functions with `Promise`

Reply:
PollRepository functions in sdk don't have callback function with return data. 
They are promises function with await from api without error catch. 
So I made Promise with reject for error log. 

Yeah, poll.sdk.ts is kind of redundant just with reject and error. we can directly use PollRepository. 
What do you think?

Reply:
we should use it directly because yeah it is redundant

Reply:
resolved krub